template-processors/openshift-provision/files/processTemplates.yml

```diff
@@ -0,0 +1,42 @@
+---
+- name: Required variable check
+  hosts: localhost
+  gather_facts: false
+
+  pre_tasks:
+    - include_vars:
+        file: "{{ config_dump }}"
+    - set_fact:
+        openshift_resource_path: []
+        openshift_cluster_provision_pre_tasks: []
+        openshift_cluster_provision_post_tasks: []
```

Comment:
should we maybe prefix them with `_`, to mark them as internal, as is done for `_cluster_definition_dir`?

Reply:
These vars are required for the openshift-provision role.
I could store them as internal vars and then explicitly set them on the actual role as that would probably be more clear.

Reply:
Oh, ok. Maybe just adding a comment above this block would be enough then?